app/team_test.go

```diff
@@ -584,9 +527,7 @@ func TestSanitizeTeam(t *testing.T) {
 		}
 
 		sanitized := th.App.SanitizeTeam(session, copyTeam())
-		if sanitized.Email == "" {
-			t.Fatal("shouldn't have sanitized team")
-		}
+		require.NotEqual(t, "", sanitized.Email, "shouldn't have sanitized team")
```

Comment:
same here and in other cases. you can use `require.Empty` and `require.NotEmpty` for `Equal` and `NotEqual` to `""`